src/config_loader.py

```diff
@@ -117,6 +117,9 @@ def get_required_env_var(var_name: str, description: str) -> str:
 # Result directory configuration
 RESULT_DIR = os.path.join(os.getcwd(), CONFIG_GENERATION.get("RESULT_DIR", "results"))
 
+# Electrical backend configuration
+ELECTRICAL_BACKEND = CONFIG_GENERATION.get("ELECTRICAL_BACKEND", "pandapower")
```

Comment:
Would be nice to have multiple options - can you add altdss with altdss_backend.py